pkg/controllers/user/alert/configsyncer/configsyncer.go

```diff
@@ -512,3 +528,29 @@ func includeProjectMetrics(projectAlerts []*v3.ProjectAlertRule) bool {
 	}
 	return false
 }
+
+func getClusterAlertGroupBy(spec v3.ClusterAlertRuleSpec) []model.LabelName {
+	if spec.EventRule != nil {
+		return eventGroupBy
```

Comment:
If you don't need to reuse the variable `eventGroupBy `, just return a new array. Same as all returns in this function.